tools-ui/src/main/java/com/hedera/hashgraph/client/ui/SettingsPaneController.java

```diff
@@ -528,7 +546,6 @@ private void setupNodeIDTextField() {
 
 		nodeIDTextField.focusedProperty().addListener((arg0, oldPropertyValue, newPropertyValue) -> {
 			if (Boolean.FALSE.equals(newPropertyValue)) {
-				logger.info("Node ID text field changed to: {}", nodeIDTextField.getText());
```

Comment:
What is the reason to remove this log but leave the other similar ones in?

Reply:
Directly after the logging the `checkNodeID` method is called that already contains enough logging from my point of view (information were doubled in the log output). Since I only looked at that workflow I can not say if it is the same for the logging that happens when the focus of the `txValidDurationTextField` chnages for example.